DDG4/src/Geant4SensitiveDetector.cpp

```diff
@@ -30,7 +30,7 @@
 #include "DD4hep/DD4hepUnits.h"
 #include "CLHEP/Units/SystemOfUnits.h"
 
-static const double MM_2_CM = (CLHEP::millimeter/dd4hep::millimeter);
+static const double MM_2_CM = (dd4hep::millimeter/CLHEP::millimeter);
```

Comment:
Shouldn't this be inside
```cpp
#ifdef HAVE_GEANT4_UNITS
#define MM_2_CM 1.0
#else
#define MM_2_CM 0.1
#endif
```

Reply:
Hi @petricm ,
I am Tao Lin from IHEP, working for CEPC. The current SD used in CEPC is still the legacy one. 

The problem we found is that in following code
```c++
    G4ThreeVector global = 0.5 * ( h.prePosG4()+h.postPosG4());
    G4ThreeVector local  = h.preTouchable()->GetHistory()->GetTopTransform().TransformPoint(global);
    Position loc(local.x()*MM_2_CM, local.y()*MM_2_CM, local.z()*MM_2_CM);
    Position glob(global.x()*MM_2_CM, global.y()*MM_2_CM, global.z()*MM_2_CM);
    VolumeID cID = seg.cellID(loc,glob,volID);
```
The units of variables `global` and `local` are `CLHEP::mm`. In order to convert to `dd4hep::mm`, we need to divide the `CLHEP::mm` first, then multiply `dd4hep::mm`.

Thank you!

Tao Lin